client/my-sites/checklist/task.jsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { noop, omit } from 'lodash';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+
+export class ChecklistTask extends PureComponent {
+	static propTypes = {
+		title: PropTypes.string.isRequired,
+		completedTitle: PropTypes.string,
+		description: PropTypes.string.isRequired,
+		duration: PropTypes.string,
+		completed: PropTypes.bool,
+		onClick: PropTypes.func,
+	};
+
+	static defaultProps = {
+		onClick: noop,
+	};
+
+	handleClick = () => {
+		this.props.onClick( omit( this.props, [ 'onClick' ] ) );
```

Comment:
Can we explicitly pass through the props that we expect the handler to have? This also prevents accidental additions if someone adds another prop in the future.